### ISSUE TITLE
prefs.js doesn't like gnome 3.38

### ISSUE THREAD

**Issue by ServerBBQ** [isjerryxiao/gnome-shell-extension-activate-gnome#2]
On gnome 3.38, the extension works as normal, except you can't access the preferences/settings.

```
TypeError: prefsWidget.append is not a function
Stack trace:
  buildPrefsWidget@/home/mayer/.local/share/gnome-shell/extensions/activate_gnome@isjerryxiao/prefs.js:29:17
  _init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
  OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
  asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:132:22
  run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
  main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
  run@resource:///org/gnome/gjs/modules/script/package.js:222:19
  start@resource:///org/gnome/gjs/modules/script/package.js:206:8
  @/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**Comment by ServerBBQ** [comment]
woops, pressed enter, will edit and reopen when done

**Comment by isjerryxiao** [comment]
https://github.com/isjerryxiao/gnome-shell-extension-activate-gnome/blob/93a41847dd29da1abce1c9387b73364a465857c5/activate_gnome@isjerryxiao/metadata.json#L6-L9
The shell-version key is there for a reason.
I don't have a gnome 3.38 desktop to test the extension with. However a PR is always welcome.

P.S. Changing `.append` to `.add` may fix the problem for gnome 3.38 but it's untested.  
see:
https://gjs-docs.gnome.org/gtk30~3.24.30/gtk.container
https://gjs-docs.gnome.org/gtk40~4.4.1/gtk.box

**Comment by isjerryxiao** [comment]
Closed due to inactivity.
